src/librustdoc/html/markdown.rs

```diff
@@ -126,26 +145,6 @@ impl<'a> Line<'a> {
     }
 }
 
-// FIXME: There is a minor inconsistency here. For lines that start with ##, we
-// have no easy way of removing a potential single space after the hashes, which
-// is done in the single # case. This inconsistency seems okay, if non-ideal. In
-// order to fix it we'd have to iterate to find the first non-# character, and
-// then reallocate to remove it; which would make us return a String.
```

Comment:
re https://github.com/rust-lang/rust/pull/74048/files#r449881015

I have no idea what this comment is talking about.
Does it replace "##" with "#" as the old code did?
So should I keep the old FIXME?

Reply:
Oh you wrote this code: ffe12b1274732e791e117d8cc79b3db0f4161956 and has a test here: https://github.com/rust-lang/rust/blob/master/src/test/rustdoc/issue-41783.rs